doc/system_specific_setup.rst

```diff
@@ -1,225 +1,266 @@
 
-   > sudo apt-get install python-pyparsing
 
-.. _install_pip:
 
-Install pip
-^^^^^^^^^^^
+Installing PSyclone
+^^^^^^^^^^^^^^^^^^^
+Change your working directory to where you would like to place the code and 
+download the latest stable release of PSyclone. 
 
-The pip tool enables Python packages to be installed from the Python
-Package Index (https://packaging.python.org/installing/). Install it like so:
-::
+.. parsed-literal::
 
-   > sudo apt-get install python-pip
+   > cd <PSYCLONEHOME>
+   > wget https://github.com/stfc/PSyclone/archive/\ |release|\ .tar.gz
+   > gunzip \ |release|\ .tar.gz
+   > tar xf \ |release|\ .tar
+   > rm \ |release|\ .tar
+   > cd PSyclone-\ |release|
+   > export PYTHONPATH=`pwd`/src:${PYTHONPATH}
+   > export PATH=`pwd`/bin:${PATH}
 
-.. _install_fparser:
+This sets up your python path and path appropriately. You may want to set
+these paths permanently (e.g. by editing your ${HOME}/.bashrc file if you run
+the BASH shell). You can also use the latest version using git, as described
+in :ref:`psyclone_from_git`.
 
-Install fparser
-^^^^^^^^^^^^^^^
+Common installation
+^^^^^^^^^^^^^^^^^^^
+To avoid warnings during the dependency intallation, it is recommended to update ``pip``
+to the latest version::
 
-fparser is also required by PSyclone but is not available from the
-Ubuntu software centre. It can instead be installed from the
-Python Package Index using pip:
-::
+    > sudo pip install --upgrade pip
 
-   > sudo pip install fparser
+.. warning::
+    As mentioned in :ref:`opensuse_user`: on OpenSUSE this commands needs to be
+    ``sudo pip2.7 install --upgrade pip``.
 
-Uninstalling is simply a matter of doing:
-::
+Next you need to install the ``fparser`` and ``pyparsing`` packages::
 
-   > sudo pip uninstall fparser
+    > sudo pip install fparser pyparsing 
 
-If you do not have sufficient privileges for a system-wide install then
-you can do:
-::
+.. tip::
 
-   > pip install --user fparser
+    With ``pip`` it is possible to install packages either system-wide
+    (which requires root privileges) as above, or for a single user only
+    (in ~/.local). While the latter is only useful for one
+    particular user, it means that PSyclone can be installed
+    using ``pip`` without needing root privileges. In order to install
+    a package for a user, add the --user command line option to
+    all pip commands. This flag requests that the packages be installed locally
+    for the current user rather than requiring root access::
 
-(The ``--user`` flag requests that the packages be installed locally for
-the current user rather than requiring root access.) In order for Python
-to find such locally-installed packages the necessary directory must be
-added to the PYTHONPATH, e.g.:
-::
+        > pip install --user fparser pyparsing
 
-   > export PYTHONPATH=/home/a_user/.local/lib/python2.7/site-packages:${PYTHONPATH}
+    If necessary, add the ``--user`` option to all ``pip`` commands and
```

Comment:
"you may remove the use of ``sudo`` and add the ``--user`` option to all ``pip`` commands described in this document."